extensions/wikia/ImageReview/ImageReviewSpecialController.class.php

```diff
@@ -28,100 +34,39 @@ protected function setGlobalDisplayVars() {
 	public function index() {
 		$this->setGlobalDisplayVars();
 
-		$action = $this->getPar();
-		$actions = explode('/',$action);
-		$action = array_pop($actions);
+		$this->action = $this->getAction();
+		$this->order = $this->getOrderingMethod();
+		$this->ts = $this->wg->request->getVal( 'ts' );
```

Comment:
it could be possible to use the local RequestContext via `->getContext()` instead of the Nirvana accessors